lua/astrocommunity/debugging/telescope-dap-nvim/init.lua

```diff
@@ -0,0 +1,60 @@
+return {
+  {
+    "nvim-telescope/telescope-dap.nvim",
+    name = "telescope_dap",
+    event = "VeryLazy",
```

Comment:
```suggestion
    event = "User AstroFile",
```
There should be a better way to lazy-load this plugin. My initial suggestion would be this unless someone has a better idea.